src/main/java/com/endava/soa_5/actions/GenericActions.java

```diff
@@ -0,0 +1,12 @@
+package com.endava.soa_5.actions;
+
+import com.endava.soa_5.page_objects.Base;
+import org.openqa.selenium.WebElement;
+
+public class GenericActions extends Base {
+    public static void clickOnElement(WebElement element){
+        element.click();
+    }
+    //add generic waiter method here
```

Comment:
Is this a TODO ? if yes can indicate it with the reference where it will be resolved